application/modules/layout/views/layout.php

```diff
@@ -44,6 +44,8 @@
 <?php echo $this->layout->load_view('layout/includes/fullpage-loader'); ?>
 
 <script defer src="<?php echo base_url(); ?>assets/core/js/scripts.min.js"></script>
+<script defer src="https://cdnjs.cloudflare.com/ajax/libs/markdown.js/0.5.0/markdown.min.js"></script>
```

Comment:
... and popular markdown parser for javascript _markdown.js_:
https://github.com/evilstreak/markdown-js